src/window/mod.rs

```diff
@@ -92,6 +101,36 @@ impl<F: Frame + 'static> Window<F> {
         shell: &Shell,
         implementation: Impl,
     ) -> Result<Window<F>, F::Error>
+    where
+        Impl: Implementation<(), Event> + Send,
+    {
+        Self::init_with_decorations(
+            surface,
+            initial_dims,
+            compositor,
+            subcompositor,
+            shm,
+            shell,
+            None,
+            implementation,
+        )
+    }
+
+    /// Create a new window wrapping a given wayland surface as its main content and using
+    /// server decorations if available
```

Comment:
You can set the client's preference to SSD if you prefer to use SSD. Right now the client preference is not set, so the compositor decides.

Reply:
My goal with that was mostly to aim for "follow the compositor's preference". But yeah, I need to clarify what policy to actually follow.